agasc/agasc.py

```diff
@@ -366,33 +406,32 @@ def get_stars(ids, agasc_file=None, dates=None, fix_color1=True, use_mag_est=Fal
     :param dates: Dates for proper motion (scalar or array) (default=Now)
     :param fix_color1: set COLOR1=COLOR2 * 0.85 for stars with V-I color (default=True)
     :param use_mag_est: Use estimated mag from AGASC supplement where available (default=False)
+    :param read_entire_agasc: Number of stars above which to read the entire AGASC (default=5000)
```

Comment:
Love that you set the number of stars as function parameter, though the name sounds more like it should be a flag/boolean.

Reply:
I kind of agree with Jean that the name suggests something else to me.

I would call it `cache_threshold` or something like that.

Reply:
Changed to `method_threshold`.  

Note - there is no caching involved, on purpose. I don't want to leave the whole AGASC and index dict in memory by default. Maybe a subsequent PR could allow that if it turns out to be useful. In my work the large `get_stars` calls are just one-offs.